### ISSUE TITLE
Behavior doesn't update when configuration settings change.

### ISSUE THREAD

**Issue by DamnedScholar** [DavidLGoldberg/jumpy#118]
I have to reload Atom every time I adjust the config settings before the package recognizes that they've changed. [Here is a GIF](https://puu.sh/Bij5m/d23d8ecb65.gif) demonstrating the phenomenon. I don't see any text in the readme or settings pane to indicate that this is expected behavior.

### Environment
Windows 10
Atom 1.29.0
Jumpy 5.0.0

**Comment by DavidLGoldberg** [comment]
Yes, at the time I thought this was acceptable and going to add a note.  Although I can "observe" the config setting:
https://atom.io/docs/api/v1.6.0/Config

Hopefully I'll get to this soon.
